README.md

```diff
@@ -294,7 +254,7 @@ behaviorService: Ember.inject.service( 'sl-behavior' ),
 
 if ( this.get( 'behaviorService' ).isAble( 'setData', 'event' ) ) { ... }
 
-if ( this.get( 'behaviorService' ).isUnable( 'setData', this.get( 'model' ) ) ) { ... }
+if ( this.get( 'behaviorService' ).isUnable( 'setData', 'event', true ) ) { ... }
```

Comment:
It could be a function, or a property/computed property, or a literal boolean. In practice it should never be a literal boolean, though, so I'll modify this example and provide some others that are more in line with how it would actually be used.